clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchFieldMapper.scala

```diff
@@ -32,34 +42,18 @@ private[dataaccess] object ElasticsearchFieldMapper
 
     override def stringToDefinition(fieldType: Type): String => FieldDefinition = {
       fieldType match {
-        case tpe if tpe =:= typeOf[Boolean]           => booleanField
-        case tpe if tpe =:= typeOf[Int]               => intField
-        case tpe if tpe =:= typeOf[Long]              => longField
-        case tpe if tpe =:= typeOf[Float]             => floatField
-        case tpe if tpe =:= typeOf[Double]            => doubleField
-        case tpe if tpe =:= typeOf[String]            => keywordField
-        case tpe if tpe =:= typeOf[Symbol]            => keywordField
-        case tpe if tpe <:< typeOf[EnumEntry]         => keywordField
-        case tpe if tpe =:= typeOf[OffsetDateTime]    => dateField
-        case tpe if tpe =:= typeOf[UpsertId]          => keywordField
-        case tpe if tpe =:= typeOf[Option[Boolean]]   => booleanField
-        case tpe if tpe =:= typeOf[Option[Int]]       => intField
-        case tpe if tpe =:= typeOf[Option[Long]]      => longField
-        case tpe if tpe =:= typeOf[Option[Float]]     => floatField
-        case tpe if tpe =:= typeOf[Option[Double]]    => doubleField
-        case tpe if tpe =:= typeOf[Option[String]]    => keywordField
-        case tpe if tpe =:= typeOf[Option[Symbol]]    => keywordField
-        case tpe if tpe <:< typeOf[Option[EnumEntry]] => keywordField
-        case tpe if tpe =:= typeOf[Option[UUID]]      => keywordField
-        case tpe if tpe =:= typeOf[Option[URI]]       => keywordField
-        case tpe if tpe =:= typeOf[Option[OffsetDateTime]] =>
-          dateField
-        /*
-         * Elasticsearch has no notion of an array field; instead, any
-         * field can hold 1+ values of the same type.
-         */
-        case tpe if tpe <:< typeOf[Option[Seq[String]]] => keywordField
-        case tpe if tpe <:< typeOf[Option[Seq[URI]]]    => keywordField
+        case tpe if super.is[Boolean](tpe)        => booleanField
+        case tpe if super.is[Int](tpe)            => intField
+        case tpe if super.is[Long](tpe)           => longField
+        case tpe if super.is[Float](tpe)          => floatField
+        case tpe if super.is[Double](tpe)         => doubleField
+        case tpe if super.is[String](tpe)         => keywordField
+        case tpe if super.is[Symbol](tpe)         => keywordField
+        case tpe if super.is[EnumEntry](tpe)      => keywordField
+        case tpe if super.is[OffsetDateTime](tpe) => dateField
+        case tpe if super.is[UpsertId](tpe)       => keywordField
+        case tpe if super.is[UUID](tpe)           => keywordField
+        case tpe if super.is[URI](tpe)            => keywordField
```

Comment:
I don't think you need `super.` for these.

Reply:
I do if it's `private`, apparently. I definitely don't if it's `protected`